dev-requirements.txt

```
@@ -1,27 +1,32 @@
+bandit==1.7.10; python_version <= "3.8"
+bandit==1.8.0; python_version > "3.8"
 black==24.10.0; python_version > "3.8"
 black==24.8.0; python_version <= "3.8"
-isort; python_version < "3.8"
-isort==5.13.2; python_version >= "3.8"
-pre-commit; python_version <= "3.8"
-pre-commit==4.1.0; python_version > "3.8"
 codespell==1.16.0
 flake8; python_version < "3.8"
 flake8==7.1.1; python_version >= "3.8"
-bandit==1.7.10; python_version <= "3.8"
-bandit==1.8.0; python_version > "3.8"
 gitlint==v0.19.1
 interrogate
+isort; python_version < "3.8"
+isort==5.13.2; python_version >= "3.8"
+jsonschema
 mypy==v1.14.1
+playwright
+pre-commit; python_version <= "3.8"
+pre-commit==4.1.0; python_version > "3.8"
 pytest>=7.2.0
-pytest-xdist
-pytest-cov
 pytest-asyncio
+pytest-cov
 pytest-mock
 pytest-playwright
-playwright
+pytest-xdist
 types-beautifulsoup4
 types-jsonschema
 types-PyYAML
 types-requests
 types-setuptools
+<<<<<<< HEAD
+types-toml
+=======
```

```suggestion
types-toml
```

Looks like you accidentally committed a file with an unresolved diff.